host/host_tensor/include/host_conv.hpp

```diff
@@ -12,9 +13,9 @@ template <typename TIn,
 void host_conv_nchw_kcyx_nkhw(const Tensor<TIn>& in,
                               const Tensor<TWei>& wei,
                               Tensor<TOut>& out,
-                              const ConvStrides& conv_strides,
-                              const ConvDilations& conv_dilations,
-                              const InLeftPads& in_left_pads,
+                              const ConvStrides& strides,
```

Comment:
Please don't change the name 

strides means a lot of things, (memory strides, convolution filter window strides)

left_pads also could mean either input tensor or output tensor left pads